pshops_lib/populate_menu.js

```diff
@@ -0,0 +1,60 @@
+/* 
```

Comment:
in this case, rather than copying over the module wholesale, just import the existing module